### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by realmbmc** [Haizanjs/rvExtension_EncryptionDLL#1]
Is it possible to decrypt the string without running this engine extension?
Also it would be cool if it would be possible to decrypt code using serverside function at client's initialization.

**Comment by Leigham** [comment]
Pretty sure it wouldn't be a very good encryption if you could decrypt it ;) you would compare a string by encrypting it, and comparing the encrypted strings.

**Comment by realmbmc** [comment]
Yeah, that's good. I've just thought about whole SQF code execution encryption. Like serverside function for decryption it using sha256 private key.

**Comment by Leigham** [comment]
the client will still the see unencrypted function so there would really be no use.

**Comment by Haizanjs** [comment]
**deleted issue**
